app/components/Form/utils.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { SubmissionError } from 'redux-form';
+import { get } from 'lodash';
+
+export const withSubmissionError = (func: any => Promise<any>) => {
+  return (...data: Array<any>) =>
+    func(...data).catch(error => {
+      const errPayload = get(error, ['payload', 'response', 'jsonData']);
+      if (!errPayload) {
+        throw error;
+      }
+
+      throw new SubmissionError({
+        ...errPayload,
+        _error: errPayload.detail
```

Comment:
A lot of django errors returns a general error as `detail`.The `_error` is sent to the form as the `error` prop, so it can be shown together with the submit button...